products/appengine/api.yaml

```
@@ -515,4 +515,70 @@ objects:
                 Pathname within the host. Must start with a "/". A single "*" can be included at the end of the path.
                 The sum of the lengths of the domain and path may not exceed 100 characters.
               required: true
+  - !ruby/object:Api::Resource
+    name: 'ServiceSplitTraffic'
+    description: |
+      Traffic routing configuration for versions within a single service. Traffic splits define how traffic directed to the service is assigned to versions.
+    base_url: 'apps/{{project}}/services'
+    create_url: 'apps/{{project}}/services/{{service_id}}?migrateTraffic={{migrate_traffic}}&updateMask=split'
+    create_verb: :PATCH
+    update_url: 'apps/{{project}}/services/{{service_id}}?migrateTraffic={{migrate_traffic}}'
+    update_verb: :PATCH
+    update_mask: true
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      api: 'https://cloud.google.com/appengine/docs/admin-api/reference/rest/v1/apps.services'
+    async: !ruby/object:Api::Async
+      operation: !ruby/object:Api::Async::Operation
```

```suggestion
      operation: !ruby/object:Api::OpAsync::Operation
```

`Async` was renamed `OpAsync`, do you mind updating that? Sorry for the delay in getting back!

---

Thanks @rileykarson.